LoRaEngine/test/LoRaWan.Test.Shared/LoRaWan.Test.Shared.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsTestProject>false</IsTestProject>
```

Comment:
👌 because otherwise we were getting some "false positives" (trying to find tests in this project and failing)

Reply:
If you don't do this, you get an error from `dotnet test` along the lines of:

    Unable to find testhost.dll. Please publish your test project and retry.

I changed the `dotnet test` invocation in the CI workflow to run all tests in the solution, with some filters applied. However, some projects, like **LoRaWan.Tests.Shared**, reference [xunit](https://www.nuget.org/packages/xunit/) and contain code to be shared by actual test projects. Unless you flag them with `IsTestProject` as `false`, `dotnet test` mistakes them for test projects and fails with the above error when trying to run them.

Reply:
I know :D that's why the 👍